src/Tribe/Repositories/Event.php

```diff
@@ -1635,7 +1669,8 @@ protected function order_by_venue() {
 		);
 
 		$filter_id = 'order_by_venue';
-		$this->filter_query->orderby( 'venue', $filter_id, true );
+		$order = Arr::get_in_any( [ $this->query_args, $this->default_args ], 'order', 'ASC' );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 5 spaces but found 1 space

<sub>hash: c51e70d193ae43929e900f086a927761</sub>